webhook/resources/backup/mutator.go

```diff
@@ -1,15 +1,20 @@
 package backup
 
 import (
+	"encoding/json"
+	"fmt"
+
 	"github.com/pkg/errors"
-	admissionregv1 "k8s.io/api/admissionregistration/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 
-	"github.com/longhorn/longhorn-manager/datastore"
 	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
-	"github.com/longhorn/longhorn-manager/webhook/admission"
 	common "github.com/longhorn/longhorn-manager/webhook/common"
 	werror "github.com/longhorn/longhorn-manager/webhook/error"
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+	"github.com/longhorn/longhorn-manager/types"
+	"github.com/longhorn/longhorn-manager/webhook/admission"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
Don't forget to update the package layout.

Reply:
Use `goimports' tool to follow imports pattern 

https://github.com/longhorn/longhorn-manager/blob/b91976a861a4ef034334b6a3f7d56a44185aa1b5/webhook/resources/backup/mutator.go#L3-L20